test/models/maintenance_tasks/run_test.rb

```diff
@@ -222,19 +222,23 @@ class RunTest < ActiveSupport::TestCase
       assert_equal 2, query_count
     end
 
-    test "#stopping? returns true if status is pausing or cancelling" do
+    test "#stopping? returns true if status is pausing or cancelling or cancelled" do
       run = Run.new(task_name: "Maintenance::UpdatePostsTask")
 
-      (Run.statuses.keys - ["pausing", "cancelling"]).each do |status|
-        run.status = status
-        refute_predicate run, :stopping?
-      end
+      (Run.statuses.keys - ["pausing", "cancelling", "cancelled"])
```

Comment:
nit: maybe this is more future-proof? 
```suggestion
      (Run.statuses.keys - Run::STOPPING_STATUSES)
```